components/engine_traits/Cargo.toml

```diff
@@ -8,6 +8,8 @@ publish = false
 failpoints = ["fail/failpoints"]
 
 [dependencies]
+bitflags = "1.3"
+codec = { path = "../codec", default-features = false }
```

Comment:
🤔

Reply:
It seems this was removed at 703d794dea95da1bc4a147d464d6b2546b17146f and reborn🤔